op.h

```diff
@@ -15,7 +15,12 @@
  *			but this is replaced when op is grafted in, when
  *			this op will point to the real next op, and the new
  *			parent takes over role of remembering starting op.)
+ * 	op_sibparent    Pointer to the op's next sibling, or to the parent
+ *			if there are no more siblings (PERL_OP_PARENT builds)
```

Comment:
All builds are PERL_OP_PARENT now, so I'm not sure it's worth mentioning.

Reply:
Ok. I'll remove that mention.